src/@batch-flask/ui/virtual-scroll/virtual-scroll.component.ts

```diff
@@ -110,7 +111,7 @@ export class VirtualScrollComponent implements OnInit, AfterViewInit, OnChanges,
     private _tail: VirtualScrollTailComponent;
     private _erd: any;
 
-    @ViewChild("padding", { read: ElementRef, static: true })
+    @ViewChild("padding", { read: ElementRef, static: false })
```

Comment:
All of those needs to be false now?

Reply:
It seems like by default it should be set to false unless if it uses `TemplateRef` in a query or the component depends on a query result to be available before ngAfterViewInit: https://angular.io/guide/static-query-migration

Reply:
I see